src/Aspire.Dashboard/Authentication/FrontendCompositeAuthenticationHandler.cs

```diff
@@ -29,13 +28,6 @@ protected override async Task<AuthenticateResult> HandleAuthenticateAsync()
                     parameters: new Dictionary<string, object?> { [AspirePolicyEvaluator.SuppressChallengeKey] = true }));
         }
 
-        var scheme = GetRelevantAuthenticationScheme();
-        if (scheme == null)
-        {
-            var id = new ClaimsIdentity([new Claim(OtlpAuthorization.OtlpClaimName, bool.FalseString)]);
-            return AuthenticateResult.Success(new AuthenticationTicket(new ClaimsPrincipal(id), Scheme.Name));
-        }
```

Comment:
So this is what didn't work with SignalR?

Reply:
Yes. It worked with the initial request (otherwise the page would have been a 401) but didn't with SignalR,. The SignalR failed then flowed into Blazor, which then caused the authorized route view component to display Unauthorized.